src/Microsoft.DocAsCode.Common/FileAbstractLayer/LinkFileReader.cs

```diff
@@ -29,7 +30,7 @@ public LinkFileReader(IEnumerable<PathMapping> mappings)
                     if (m.AllowMoveOut || localPath.ParentDirectoryCount == 0)
                     {
                         var physicalPath = Path.Combine(m.PhysicalPath, localPath.ToString());
-                        if (File.Exists(physicalPath))
+                        if (File.Exists(Environment.ExpandEnvironmentVariables(physicalPath)))
```

Comment:
saving with initial %env% while getting with expand?
feels not save to let the caller make the expanding